projects/smoke_adt7476a/src/main.c

```diff
@@ -0,0 +1,76 @@
+#include "adt7476a_fan_controller.h"
+#include "adt7476a_fan_controller_defs.h"
+#include "gpio.h"
+#include "i2c.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+#define SET_SPEED_INTERVAL 5
```

Comment:
please add a unit suffix at the end, e.g. `_S`. Also minor nit, the default interval should probably be 1 second so it's more immediately obvious if it's working or not